packages/cli/src/util/dev/server.ts

```diff
@@ -1740,7 +1741,10 @@ export default class DevServer {
             isDev: true,
             requestPath,
             devCacheDir,
-            env: { ...envConfigs.runEnv },
+            env: {
+              ...envConfigs.runEnv,
+              VERCEL_BUILDER_DEBUG: this.output.debugEnabled ? '1' : undefined,
```

Comment:
This allows `debug` function calls (from build-utils) to log or not inside the spawned process